tests/integration/guzzle/test/VCR/Example/ExampleHttpClientTest.php

```diff
@@ -20,19 +24,18 @@ class ExampleHttpClientTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        // Configure virtual filesystem.
-        vfsStream::setup('testDir');
-        \VCR\VCR::configure()->setCassettePath(vfsStream::url('testDir'));
+        \VCR\VCR::configure()->setCassettePath('/tmp');
```

Comment:
We should still use vfsStream. In theory this shouldn't be a problem to use. Writing to the file system is not handy for repeatable tests since the fixture file then needs to be deleted.